src/match.c

```diff
@@ -1,128 +1,209 @@
 #include <stdio.h>
 #include <stdlib.h>
-#include <stdbool.h>
-#include <assert.h>
-#include <gmodule.h>
+#include "simclist.c"
+#include "simclist.h"
 #include "match.h"
 
 /* See match.h for descriptions of functions */
 
-match* new_match(char* word, int line)
```

Comment:
Every function in this file should start with `match_`.

Reply:
fixed